canvas_modules/common-canvas/src/command-actions/deleteObjectsAction.js

```diff
@@ -172,16 +172,29 @@ export default class DeleteObjectsAction extends Action {
 	}
 
 	createActionLabel() {
+
+		const stringList = [
+			{ label: "Nodes", val: this.nodesToDelete.length + this.supernodesToDelete.length },
+			{ label: "Comments", val: this.commentsToDelete.length },
+			{ label: "Links", val: this.linksToDelete.length }
+		];
+		const dynamicStringKey = "action.delete" + stringList
+			.filter((v) => v.val).map((o) => o.label)
```

Comment:
Just for readability, please move the `.map((o) => o.label)`  onto a separate line.